cmd/onboard.go

```diff
@@ -58,12 +65,54 @@ Use deal templates to configure deal parameters - individual deal flags are not
 		},
 		&cli.StringSliceFlag{
 			Name:     "source",
-			Usage:    "Local source path(s) to onboard",
+			Usage:    "Source path(s) to onboard (local paths or remote URLs like s3://bucket/path)",
 			Required: true,
 		},
 		&cli.StringSliceFlag{
 			Name:  "output",
-			Usage: "Local output path(s) for CAR files (optional)",
+			Usage: "Output path(s) for CAR files (local paths or remote URLs like s3://bucket/path)",
+		},
+
+		// Remote storage configuration
+		&cli.StringFlag{
+			Name:  "source-type",
+			Usage: "Source storage type (local, s3, gcs, azure, etc.)",
+			Value: "local",
+		},
+		&cli.StringFlag{
+			Name:  "source-provider",
+			Usage: "Source storage provider (for s3: aws, minio, wasabi, etc.)",
+		},
+		&cli.StringFlag{
+			Name:  "output-type",
+			Usage: "Output storage type (local, s3, gcs, azure, etc.)",
+			Value: "local",
+		},
+		&cli.StringFlag{
+			Name:  "output-provider",
+			Usage: "Output storage provider",
```

Comment:
This is not resolved, Add that this will default to the same type as the output-type if left blank.